src/android/webview.kt

```diff
@@ -116,22 +167,23 @@ export default module
         "Access-Control-Allow-Methods" to "*"
       )
 
+      // prevent piped streams blocking each other, have to write on a separate thread if data > 1024 bytes
       val bytes = moduleTemplate.toByteArray()
-      stream.write(bytes, 0, bytes.size)
-      stream.close()
+      kotlin.concurrent.thread {
+        stream.write(bytes, 0, bytes.size)
```

Comment:
The code was previously like this, the only change on this branch is to use `kotlin.concurrent.thread`. 

At one point I was sending preload js via this route, and I got stuck with thread lockups due to buffers > 1024 bytes.

I think it doesn't hurt to have `kotlin.concurrent.thread` there in case someone makes a similar modification in the future, thoughts?